app/views/student/messages/index.html.haml

```diff
@@ -0,0 +1,13 @@
+.container.h-100
+  %h1.py-4.text-dark オリアプ相談チャット
+  .h-100.my-3.border
+    .d-flex.flex-column.px-0
+      .list-group.w-100.px-1.py-1#messages
+        = render partial: 'shared/message', collection: @messages
+      .mt-auto.w-100.px-2.py-1.bg-secondary
+        = form_for ['student', @message] do |f|
+          .d-flex.justify-content-between
+            = f.hidden_field :chat_group_id, value: current_student.id, id: 'chat_group_id'
```

Comment:
__[HAML-Lint]__

Line is too long. [91/80]


<!-- Comment by [SideCI](https://sideci.com) -->